teku/src/main/java/tech/pegasys/teku/cli/options/ValidatorOptions.java

```diff
@@ -81,6 +81,18 @@
       arity = "1")
   private Bytes32 graffiti;
 
+  @Option(
+      names = {"--validator-performance-tracking-enabled"},
+      paramLabel = "<BOOLEAN>",
+      description = "Enable validator performance tracking and logging",
+      fallbackValue = "true",
+      arity = "0..1")
+  private boolean validatorPerformanceTrackingEnabled = false;
```

Comment:
We should probably add a few tests for this to `ValidatorOptionsTest`

Reply:
Done.